packages/web-components/src/textarea/textarea.ts

```diff
@@ -275,7 +274,7 @@ export class BaseTextArea extends FASTElement {
       toggleState(this.elementInternals, `resize-${next}`, true);
     }
 
-    toggleState(this.elementInternals, `resize`, TextAreaResizableResize.includes(this.resize));
+    toggleState(this.elementInternals, `resize`, this.resize !== TextAreaResize.none);
```

Comment:
Maybe we should check if `this.resize` is either `TextAreaResize.horizontal` or `TextAreaResize.vertical` to avoid false positives?

Reply:
I think it can also be `TextAreaResize.both`?

Reply:
Yes and that.

Reply:
@marchbox okay! updated.